client/src/components/More.jsx

```diff
@@ -110,7 +136,17 @@ const More = () => {
         );
       })
       .catch(err => {
-        console.log(err);
+        let errorMessage = '';
```

Comment:
Could we have a higher-level `errorMessage` variable so as to not declare it twice?